rust_src/src/buffers.rs

```diff
@@ -118,6 +120,11 @@ impl LispBufferRef {
         LispObject::from(self.name)
     }
 
+    #[inline]
+    pub fn is_read_only(&self) -> LispObject {
```

Comment:
Is it worth simplifying the Rust code to have this return a bool?

If nil false else true?

Reply:
@brotzeit ? Is there are reason to have this return a LispObject?